ngraph/frontend/onnx_import/src/onnx.cpp

```diff
@@ -74,7 +74,7 @@ namespace ngraph
             } // namespace error
 
             static const std::vector<std::string> legacy_ops_to_fixup = {
-                "FakeQuantize", "DetectionOutput", "Normalize", "PriorBox"};
+                "DetectionOutput", "FakeQuantize", "GroupNorm", "Normalize", "PriorBox"};
```

Comment:
@postrational, finally, have you found a reliable description for all those operations?